cloud/deploys/azure/bin/db-connection

```diff
@@ -0,0 +1,27 @@
+# example: ./db-connection -g sgdev -d civiform-artistic-halibut -k bastion
+while getopts g:d:k flag
+do
+    case "${flag}" in
+        g) resource_group=${OPTARG};;
+        d) database_name=${OPTARG};;
+        k) key_name=${OPTARG};;
+    esac
+done
+ssh-keygen -t rsa -b 4096 -f $HOME/.ssh/${key_name}
```

Comment:
question (blocking): i wonder if it would be better to store the key in the project root directory and instruct the user to destroy it after they're done? thoughts?

Reply:
yeah i think maybe the other option would be to check if it exists then if it does already just skip the generation step. not sure if it really matters if the key gets deleted

Reply:
alright i did the like skip if it already exists option

Reply:
and i also destory the key afterwards so i will resolve this